.github/workflows/prettier.yml

```diff
@@ -0,0 +1,53 @@
+name: Prettier
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches:
+      - main
+
+  pull_request:
+    branches:
+      - main
```

Comment:
Out of interest,  does this restrict the destination branch to main or the source branch? Prettier plugin is failing for me, claiming it's in detached head state...